AEPAnalytics/Sources/Analytics.swift

```diff
@@ -136,8 +157,13 @@ public class Analytics: NSObject, Extension {
             ret[AnalyticsConstants.AnalyticsRequestKeys.PAGE_NAME] = stateName
         }
 
-        // Todo:- Aid. Should we add it to identity map or vars
-        // Todo:- Vid. Should we add it to identity map or vars
+        if let aid = getAID() {
```

Comment:
Do we need to handle ignore aid value too?

Reply:
No. We don't use it here. I am migrating all the values so that we can reuse the same class in analytics direct extension.